guides/source/6_0_release_notes.md

```diff
@@ -641,10 +641,196 @@ Please refer to the [Changelog][active-support] for detailed changes.
 
 ### Removals
 
+*   Remove deprecated `#acronym_regex` method from `Inflections`.
+    ([Commit](https://github.com/rails/rails/commit/0ce67d3cd6d1b7b9576b07fecae3dd5b422a5689))
+
+*   Remove deprecated `Module#reachable?` method.
+    ([Commit](https://github.com/rails/rails/commit/6eb1d56a333fd2015610d31793ed6281acd66551))
+
+*   Remove `` Kernel#` `` without any replacement.
```

Comment:
What's with the backticks on this line?

Reply:
This is to make sure that the backtick in the method name gets rendered correctly.

![image](https://user-images.githubusercontent.com/621238/57055394-d08a0800-6cb8-11e9-91bf-639b69c18202.png)